package.json

```diff
@@ -11,7 +11,7 @@
     "test:coverage": "cross-env BABEL_ENV= NODE_ENV=test nyc --reporter=lcov --reporter=text-summary mocha --require @babel/register \"tests/**/*.tests.js\"",
     "test:watch": "npm test -- --watch",
     "docs": "jsdoc --readme README.md ./lib/*.js -d ./docs && jsdoc2md ./lib/*.js > ./api.md",
-    "build": "rm -rf ./dist/** && babel lib --out-dir ./dist --ignore *.tests.js --source-maps",
+    "build": "rm -rf ./dist/** && rollup -c && babel ./dist/index.js --out-dir ./dist --source-maps",
```

Comment:
Please create an own command `build:merge` for merging the files. Here is why

- commands should be atomic; one command for one action (similar to writing functions - think of [SRP](https://en.wikipedia.org/wiki/Single-responsibility_principle))
- in case we want to replace rollup with something else, we only need to change the `build:merge` command and can keep any other commands as they are
- we can build different chains with atomic commands, depending on our situation or environment